test-runner/run-tests.sh

```diff
@@ -2,6 +2,7 @@
 
 # Our test suite config
 CONTROLLER_HOST="http://iofog-controller:51121"
+CONNECTOR_HOST="http://iofog-connector:8080"
```

Comment:
Currently, the smoke testing suite doesn't support pointing at two different hosts. Therefore, this won't have any effect anywhere. We need to augment the framework to handle multiple urls. Something like this:

https://github.com/svanoort/pyresttest/issues/187